front/app/containers/Authentication/steps/LightFlowStart/SSOButtons/index.tsx

```diff
@@ -80,19 +44,22 @@ const SSOButtons = (props: Props) => {
         </Box>
       )}
       <Box>
-        {googleLoginEnabled && <SSOButton ssoProvider="google" {...props} />}
```

Comment:
This was the main issue that I was trying to solve, before I decided to refactor this whole mess. Basically, when adding new SSO methods, people were supposed to add them to this file. But there are two things they needed to do: 1. add the feature flag, 2. add the button. For some very strange reason everyone decided to only add the feature flag, and not the button. And made the same mistake like 7 times. What the hell